gitlab/resource_gitlab_project_cluster.go

```diff
@@ -0,0 +1,245 @@
+package gitlab
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/xanzy/go-gitlab"
+	"log"
```

Comment:
please put go internal deps first, then new line , then github deps

Reply:
done in 887f66b